modules/administration-guide/pages/configuring-advanced-authorization.adoc

```diff
@@ -0,0 +1,54 @@
+:_content-type: PROCEDURE
+:description: Configuring advanced authorization
+:keywords: authorization, user, group
+:navtitle: Configuring advanced authorization
+// :page-aliases:
+
+[id="configuring-advanced-authorization"]
+= Configuring advanced authorization
+
+You can determine which users and groups are allowed to access {prod-short}.
+
+.Prerequisites
+
+* An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster. See {orch-cli-link}.
+
+.Procedure
+
+. Configure the `CheCluster` Custom Resource. See xref:using-the-cli-to-configure-the-checluster-custom-resource.adoc[].
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+spec:
+  networking:
+    auth:
+      advancedAuthorization:
+        allowUsers:
+          - __<allow_users>__ <1>
+        allowGroups:
+          - __<allow_groups>__ <2>
+        denyUsers:
+          - __<deny_users>__ <3>
+        denyGroups:
+          - __<deny_groups>__ <4>
+----
+<1> List of users allowed to access {prod}.
+<2> List of groups of users allowed to access {prod} (for {ocp} only).
+<3> List of users denied access to {prod}.
+<4> List of groups of users denied to access {prod} (for {ocp} only).
++
```

Comment:
ok, looks liks on k8s one can only use `allowUsers` / `denyUsers`, whereas `allowGroups` / `denyGroups` is OpenShift specific.